src/PathItem.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const PathItem = ({pathName, pathItem}) => {
```

Comment:
The `return` is implicit without curly braces, but required if you use them. So you can use what you have now, or switch to:

```javascript
const PathItem = ({pathName, pathItem}) =>
  <div>
    <h2>{pathName}</h2>
    {/* render Operations(s) */}
  </div>
```